all-in-one/dockerhub-description.yml

```diff
@@ -0,0 +1,23 @@
+# This uses the peter-evans workflow: https://github.com/peter-evans/dockerhub-description
+name: Update Docker Hub Short-Description & Overview
+on:
+  push:
+    branches:
+      - main
```

Comment:
Hey! Thinking about this more, is there a specific reason why we're going with this trigger and using a separate workflow from https://github.com/EasyDynamics/oscal-editor-deployment/blob/main/.github/workflows/container-test-deploy.yml and adding it as a step with a "if not a pull request" condition?

I think both are fine options but I think we should definitely understand which we're choosing here and what each means.

So the existing workflow is triggered on pull requests, on all pushes to `main`, and can be triggered manually. When **not** a pull request, it pushes the updated image to Docker Hub and GitHub Container Registry. This workflow _only_ happens on pushes to `main` when specific paths are modified. Perhaps that makes it somewhat more efficient but logically it may feel easier to be able to say "the description is updated every time the image is" rather than having to manage these workflows separately.

What do you think?

Reply:
I'm leaning more towards the stance of grouping this workflow with the rest (in `container-test-deploy.yml`). Although we will update the description less often, it seems to make more sense to have the description update whenever the image is.